modules/ROOT/pages/ftp/ftp-documentation.adoc

```diff
@@ -65,27 +71,27 @@ Connects to an FTP server
 ** SECONDS
 ** MINUTES
 ** HOURS
-** DAYS |  +++A TimeUnit which qualifies the #responseTimeoutUnit attribute. Defaults to SECONDS+++ |  +++SECONDS+++ | {nbsp}
-| Host a| String |  +++The FTP server host, such as www.mulesoft.com, localhost, or 192.168.0.1, etc+++ |  | *x*{nbsp}
-| Port a| Number |  +++The port number of the FTP server to connect+++ |  +++21+++ | {nbsp}
-| Username a| String |  +++Username for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Password a| String |  +++Password for the FTP Server. Required if the server is authenticated.+++ |  | {nbsp}
-| Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | {nbsp}
-| Pooling Profile a| <<PoolingProfile>> |  +++Characteristics of the connection pool.+++ |  | {nbsp}
-|===
-
-==== Associated Operations
-* <<copy>> {nbsp}
-* <<createDirectory>> {nbsp}
-* <<delete>> {nbsp}
-* <<list>> {nbsp}
-* <<move>> {nbsp}
-* <<read>> {nbsp}
-* <<rename>> {nbsp}
-* <<write>> {nbsp}
+** DAYS |  A *Time unit* that qualifies the *Response timeout unit* attribute.  |  `SECONDS` | 
+| Host a| String |  The FTP server host, such as `www.mulesoft.com`, `localhost`, or `192.168.0.1`, and so on. |  | x
+| Port a| Number |  The port number of the FTP server to which to connect. |  `21` | 
+| Username a| String |  Username for the FTP Server. Required if the server is authenticated. |  | 
+| Password a| String |  Password for the FTP Server. Required if the server is authenticated. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+| Pooling Profile a| <<PoolingProfile>> |  Characteristics of the connection pool. |  | 
+|===
+
+== Supported Operations
```

Comment:
Move the list of operations and sources to be under the == Operations heading. Then delete the Supported Operations heading